src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.Stream.cs

```diff
@@ -12,6 +12,16 @@ namespace System.Text.Json.Serialization
 {
     public static partial class JsonSerializer
     {
+        /// <summary>
+        /// Read from a UTF-8 encoded <see cref="System.IO.Stream"/> and return the converted value.
+        /// </summary>
+        /// <returns>The converted value.</returns>
+        /// <param name="utf8Json">The UTF-8 encoded <see cref="System.IO.Stream"/>.</param>
+        /// <param name="options">The <see cref="JsonSerializerOptions"/> used to read and convert the JSON.</param>
+        /// <param name="cancellationToken">The <see cref="System.Threading.CancellationToken"/> which may be used to cancel the read operation.</param>
```

Comment:
nit: we could just borrow from existing docs for this. I am fine with how its written atm though.

https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.readasync?view=netframework-4.7.2
> The token to monitor for cancellation requests. The default value is None.

@stephentoub, do you have any particular thoughts on the wording of this param?